JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -71,7 +71,7 @@ module Microsoft.ApplicationInsights.Context {
                 this.initializeAutomaticSession();
             }
 
-            var now = +new Date;
+            var now = dateTime.Now();
```

Comment:
You will also have code that only runs in the window context. In the worker context, you won't have a navigationStart so you'll need another technique for determining when the timeOrigin (aka timing.navigationStart) is.